batch2/batch/front_end/front_end.py

```diff
@@ -226,7 +221,7 @@ def create_job(app, jobs_builder, batch_id, userdata, job_spec):  # pylint: disa
     jobs_builder = JobsBuilder(app['db'])
     try:
         for job in jobs:
-            create_job(app, jobs_builder, batch.id, userdata, job)
+            create_job(app, jobs_builder, batch.id, job)
```

Comment:
can you rename job to job_spec or spec? It's confusing when job to me means a Job object.

Reply:
Fixed.  I did an aggressive bit of consistent renaming.